src/pymodaq/daq_utils/h5modules.py

```diff
@@ -1881,37 +1897,84 @@ def add_detector(self, det_name, settings):
                                      title='Time axis',
                                      metadata=dict(label='Time axis', units='s', nav_index=0))
 
-    def add_datas(self, datas):
-        det_name = datas['name']
+    def add_datas(self, acquisition):
```

Comment:
No typehint here for `acquisition`?

Reply:
Done.

Ok but renaming method will be done maybe latter sounds a bit risky ;)

Reply:
data is already plural, you're right. But it is kind of an early mistake I propagated everywhere in the code...